packages/mongoose/src/config.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * © Copyright 2020 HP Development Company, L.P.
+ * SPDX-License-Identifier: MIT
+ */
+
+export default {
+    ALLOW_ESTIMATED_DOCUMENT_COUNT: process.env.ALLOW_ESTIMATED_DOCUMENT_COUNT || false
```

Comment:
instead of using env variables, let's add a configuration object to the [createMongooseController](https://github.com/HPInc/davinci/blob/master/packages/mongoose/src/createMongooseController.ts#L45-L48) function signature.
I'd also change it to use the estimated document count by default (introducing a breaking change).
```ts
interface AdditionalOptions {
   useEstimatedDocumentCount: boolean
}

export const createMongooseController = <T extends Constructor<{}>>(
	Model,
	ResourceSchema,
    options: AdditionalOptions
): Constructor<IMongooseController> & T => {
const { useEstimatedDocumentCount } = { useEstimatedDocumentCount: true, ...options };
```

About the breaking change, there is a specific commit structure to follow to make sure the version of the package will be changed semantically:
```
feat: introduced useEstimatedDocumentCount parameter

BREAKING CHANGE: now using the estimated document count by default
```

Reply:
@adriasala124 your [commit](https://github.com/HPInc/davinci/pull/122/commits/104c6db419585d0c66e58b0e37a5e37a4fee18b3) doesn't follow the correct notation, it must be a multiline commit, exactly as I showed above

Reply:
Sorry! It should be ok now :) Also I made the options optional. It will still default to true as it is defined in the body, but in this way we don't need to modify the tests.

Reply:
yes, looks correct now :+1: